python/tchannel/tornado/connection.py

```diff
@@ -445,7 +425,7 @@ def senderror(self, code, message, message_id):
 
         self._write(
             ErrorMessage(
-                code=ErrorCode.bad_request,
+                code=code or ErrorCode.bad_request,
```

Comment:
`code` doesn't have a default value. In fact, we explicitly check that it's in the list of accepted error codes 5 lines above. When do you expect it to be unset?